UCM-Powershell/UCM-SQL-Monitor.ps1

```diff
@@ -61,10 +64,23 @@ ForEach ($server in $SQLServers.ChildNodes)
   $job = Start-Job -filepath "$PSScriptRoot\Monitor-Worker.ps1" -ArgumentList $arguments
   Write-Log "Spawning worker thread : $($job|out-string)"
 }
+$SleepTime = 0
 while($TRUE)
 {
   $jobs = Get-job
-  Write-Host "Running $($jobs.count) workers. Check copperegg-metrics.log for any worker logs.
-              Press Ctrl+C to stop monitoring"
-  Start-Sleep -s 60
+  $StopJob = Test-Path -Path "$env:temp\stop-ucm-monitor.txt"
+  if ($StopJob)
+  {
+    Write-Log "Stopping job as requested by user."
+    Remove-Item "$env:temp\stop-ucm-monitor.txt"
+    exit 0
+  }
+  if ($SleepTime % 60 -eq 0)
+  {
+    Write-Host "Running $($jobs.count) workers. Check copperegg-metrics.log for any worker logs."
```

Comment:
What is we are starting script and the file is present ? The script will kill itself.